android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -853,6 +862,15 @@ sealed class CookieBannerAction : BrowserAction() {
      */
     data class UpdateStatusAction(val tabId: String, val status: CookieBannerHandlingStatus) :
         CookieBannerAction()
+
+    /**
+     * Updates the [SessionState.reportSiteDomainSnackBarVisibility] state.
+     */
+    data class UpdateReportSiteDomainSnackBarVisibility(
+        val tabId: String,
+        val reportSiteDomainSnackBarVisibility: Boolean,
+    ) :
+        CookieBannerAction()
```

Comment:
This is app's specific (UI), we shouldn't add it as part of `BrowserAction` which is something generic to all apps (ac), this should be on the app layer.

Reply:
Done